lib/philips.js

```diff
@@ -52,21 +52,14 @@ const knownEffects = {
 
 const extend = {
     light_onoff_brightness: (options={}) => {
-        options = {disableHueEffects: true, ...options};
-        if (!options.disableHueEffects) options.disableEffect = true;
         const result = extendDontUse.light_onoff_brightness(options);
         result['ota'] = ota.zigbeeOTA;
         result['meta'] = {turnsOffAtBrightness1: true};
         result['toZigbee'] = result['toZigbee'].concat([philipsTz.hue_power_on_behavior, philipsTz.hue_power_on_error]);
-        if (!options.disableHueEffects) {
```

Comment:
I don't think this should be removed, e.g. [`046677562229`](https://github.com/Koenkk/zigbee-herdsman-converters/blob/3656a900508768a935fdf0d5bf122ff42c01a5cf/devices/philips.js#L2793) is known to support the effects>

Reply:
So do we keep onoff_brightness with the old default or do we also flip it? The few random ones i checked were not in the pdf vs all of the colortemp and color ones which wete?

Reply:
👍 will give this another pass tomorrow